src/main/java/com/amazon/aocagent/models/Context.java

```diff
@@ -59,4 +59,18 @@
   /** Test resources region. */
   private String region;
 
+  /** EKS cluster attributes. */
+  private String eksEndpoint;
+
+  /** EKS cluster certificate. */
+  private String eksCertificate;
```

Comment:
what's this cert used for?

Reply:
https://docs.aws.amazon.com/eks/latest/APIReference/API_Certificate.html